date_and_time/task2.js

```diff
@@ -0,0 +1,2 @@
+const date = new Date();
+console.log('Дата: ' + date.getDate() + '.' + date.getMonth() + '.' + date.getFullYear() + 'г.')
```

Comment:
date.getDate() + 1, because months is counted from 0

Reply:
fixed! Forgot about it)

Reply:
well, I made a mistake. You should make `date.getMonth()` + 1, not `date.getDate()`.

Reply:
Yeah, I understand. Fixed it